README.md

```diff
@@ -1,20 +1,47 @@
-This repo contains a list of Engineering titles, roles, and associated responsibilities:
+The Mariana Tek Engineering Ladder defines the responsibilities and expectations of people working at various levels within the engineering team. It exists as a guide for engineers pursuing excellence and advancement, as well as a framework for managers to evaluate performance and identify candidates for promotion.
 
+Like many engineering teams, we outline two paths for career advancement: the management track and the technical track. Engineers pursuing the management track ("Managers") are interested in managing people and teams. Engineers pursuing the technical track ("Makers") are interested in honing their technical expertise and shaping the technical direction of the company.
+
+Framed another way by engineering team at [Chartbeat](https://engineering.chartbeat.com/2015/06/05/engineering-ladders/), the difference between Makers and Managers is how they answer this question:
+
+> “Do I want to build bigger and better systems, or do I want to manage bigger and better teams?”
+
+Every engineer starts their career as a maker. As you work your way from Engineer I to Engineer II to Senior Engineer, you are primarily focused on expanding your technical knowledge and mastering the skills required of a professional software engineer. When you're ready to move beyond Senior Engineer, you have the opportunity to specialize.
```

Comment:
I'm for the nomenclature here, but as per our discussions - there's just a gap between what we titling between systems. I don't think this is a hard block, because - frankly - our titles (as with many teams) have never cleanly aligned with any sort of rigid level stratification.

In as much as the `I` and `II` provide some space for us acknowledge junior, mid-level, and senior titles formally.  What discrepancy may exist on paper or Paylocity can be (ideally and I think in large part) addressed via other avenues of review.

Reply:
Would you prefer to change `Engineer I` to `Junior Engineer` and `Engineer II` to `Mid-Level Engineer` or `Engineer`?

Reply:
I'm not sure I fully understand Marc's comments and hope he will be able to elaborate. Did you mean that there has always been a level of ambiguity with titles for better or for worse? Are you suggesting not having these titles so much an reinforcing them through these "other avenues of review"?

FWIW, I prefer `Engineer I` and `Engineer II`. I am curious what @Asilo5 prefers?

Reply:
I like `Engineer I` and `Engineer II`. I think it states what stage the developer is or will be starting at pretty well, and the growth from that can be understood throughout the time.

Reply:
Definitely a strict preference for `I` and `II` here - would push back against using junior or mid-level at all costs personally.

@tracyalison11 - my points around title discrepancy here are twofold, really:

- (1) just acknowledging that titles were all over the place in the early days of Mariana Tek (not uncommon of course in our industry) 
- (2) emphasizing that while a level-system (whether its the one we're all in agreement on here with `I` and `II`, etc or something like `junior-`, `mid-level-`, etc in other companies) will hopefully provide additional clarity for both existing and future team members, we may still expect to see a gap in how titling is presented across systems (i.e., one's title may appear in a corporate HR system differently than how it might appear in an employee review document, etc)

The takeaway here ideally is that even if the Paylocity system of record or Workplace may not have the level of granularity we'd hope for - these titles hopefully become the standard by which we use in reviews, promotions, etc.

Hopefully that makes sense to all!

Reply:
@bashhack do we know if there are plans to align the titles across TSG? Even if just across the boutique vertical? While it may not be possible to align titles in every system (Paylocity for example), I see value in aligning our titles across the Boutique brands.